integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/createTable/TestNonTransactionalCarbonTable.scala

```diff
@@ -2490,6 +2490,54 @@ class TestNonTransactionalCarbonTable extends QueryTest with BeforeAndAfterAll {
     FileUtils.deleteDirectory(new File(writerPath))
   }
 
+  test("check varchar with trailing space") {
```

Comment:
No need to duplicate test cases. In the existing varchar columns test case, add a trailing space to one of the columns.

Reply:
besides, this is for varchar columns, why not update the code there?